.travis.yml

```diff
@@ -12,17 +13,23 @@ env:
 - ARDUINO_VERSION=nightly ARDUINO_ESP8266_VERSION=master ARDUINO_ROOT=${HOME}/arduino-${ARDUINO_VERSION} ARDUINO_ESP8266_ROOT=${ARDUINO_ROOT}/hardware/esp8266com/esp8266 ARDUINO_HOME=${HOME}/Arduino
 install:
 - if [ "$CXX" = "g++" ]; then export CXX="g++-4.8" CC="gcc-4.8"; fi
-- ( cd ${HOME} && curl -O https://downloads.arduino.cc/arduino-${ARDUINO_VERSION}-linux64.tar.xz && tar xvf arduino-${ARDUINO_VERSION}-linux64.tar.xz )
+- ( cd ${HOME} && curl -O https://downloads.arduino.cc/arduino-${ARDUINO_VERSION}-linux64.tar.xz && tar xf arduino-${ARDUINO_VERSION}-linux64.tar.xz )
 - ( cd ${HOME} && wget -q -O SoftwareSerial.zip https://github.com/plerup/espsoftwareserial/archive/097712eb07f5b3a70ef419b6e7a7ed2ada5aab85.zip && unzip -q SoftwareSerial.zip
   && rm -rf SoftwareSerial.zip && mv espsoftwareserial-* SoftwareSerial)
+- (cd ${HOME} && LIB=Adafruit_NeoPixel && wget https://github.com/adafruit/$LIB/archive/master.zip -O $LIB.zip && unzip $LIB.zip && mv $LIB-master $LIB)
```

Comment:
are we sure we want master? it would be nice to use env var and target the stable version and master in different build.